spec.html

```diff
@@ -31214,7 +31213,7 @@ <h1>Alternative</h1>
                 1. Return _m2_(_y_, _c_).
               1. Return _m1_(_x_, _d_).
           1. Else,
-            1. Assert: _direction_ is equal to -1.
+            1. Assert: _direction_ = -1.
```

Comment:
this reads a bit confusing to me, since it's in a code-like location and not in an "if". `is` would be clearer to me.

Reply:
Changed this one to `is`.